src/components/Book.js

```diff
@@ -1,27 +1,58 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/booksSlice';
+import './Book.css';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   const onRemoveClicked = (id) => {
     dispatch(deleteBook(id));
   };
   return (
-    <div>
-      <h1>{book.title}</h1>
-      <h2>
-        Author:
-        {book.author}
-      </h2>
-      <button onClick={() => onRemoveClicked(book.item_id)} type="button">Delete</button>
+    <div className="main-book" key={book.item_id}>
+      <div>
+        <p className="category">{book.category}</p>
+        <h1 className="Title">{book.title}</h1>
+        <p className="author">
+          Author:
+          {' '}
+          {book.author}
+        </p>
+        <div className="btn">
+          <button type="button" className="comments">Comments</button>
+          <div className="vertical" />
+          <button
+            type="button"
+            className="comments"
+            onClick={() => onRemoveClicked(book.item_id)}
+          >
+            Remove
+          </button>
+          <div className="vertical" />
+          <button type="button" className="comments">Edit</button>
+        </div>
+      </div>
+      <div className="circle">
+        <div className="oval" />
+        <div>
+          <h1 className="percentage">64%</h1>
+          <p className="Completed">Complete</p>
+        </div>
+      </div>
+      <div className="last">
+        <span className="Chapter">Current Chapter</span>
+        <span className="Lesson">Chapter 17</span>
+        <div className="update">Update Progress</div>
```

Comment:
- [ ] I appreciate your dedication to achieving a design-compliant project. As mentioned earlier, it's crucial to update your code to match the specified font styles, colors, and gaps outlined in the design. By making these adjustments, you'll ensure that your project closely aligns with the intended design, creating a more visually consistent and polished user interface.

Keep up the excellent work, and thank you for your commitment to adhering to the design guidelines! 🌟🎨